### DISCUSSION TITLE
Wikimedia, translatewiki.net and translations for VideoJS

### DISCUSSION THREAD

**Discussion by In1quity** [videojs/video.js#8004] [Ideas]
Hi all! My name is Iniquity and I am one of the contributors to the Wikimedia projects.

As you may know, at the beginning of this year MediaWiki abandoned the previous player and switched to VideoJS. I want to express my gratitude to you for creating this player, it is very cool :)

And here we have one suggestion / problem / wish. Wikimedia is an international movement and our platforms are typically translated into over 300 languages. And we really want your project to join our translation scheme too.

We will kill two birds with one stone, you will receive translation into many languages and we will receive it after. What do you think about this?

To translate software, we use a project called https://translatewiki.net/
How to attach your project can be found here: https://translatewiki.net/wiki/Translating:New_project/ru
Downstream task: https://phabricator.wikimedia.org/T311343

**Comment by misteroneill** [comment]
@In1quity Thanks for reaching out on this and sorry for the delayed response! This is indeed an interesting idea and we will read up on it and get back to you.

**Reply by In1quity** [reply]
> @In1quity Thanks for reaching out on this and sorry for the delayed response! This is indeed an interesting idea and we will read up on it and get back to you.

Hello! Any updates on this? :)

**Reply by In1quity** [reply]
I apologize for reminding you, but it is very important for us to know what your plans are :)

**Comment by In1quity** [comment]
Hi! I'm here again :) If the issue is the relevance and quality of translations, then you can create a setting that allows you to choose which translation to use: from translatewiki.net or manual.
